server/server/data_loader/po.py

```diff
@@ -194,3 +200,33 @@ def load_po_texts(change_tracker, po_dir, db):
                     doc['_key'] = f'{doc["lang"]}_{doc["uid"]}_{doc["author"]}'
                     collection = db.collection('po_strings')
                     insert_or_update(collection, doc)
+
+    generate_htmls(db)
+
+
+def generate_html(markup: str, translation: List[List[str]]) -> str:
+    translator = {string[0]: string[1] for string in translation}
+
+    pattern = re.compile('|'.join(re.escape(f'{key}">') for key in translator.keys()))
```

Comment:
it's not immediately obvious what's happening here, do you think we should add comments / docstrings?